src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -123,15 +131,15 @@ public string ToSqlString(ISqlDialect dialect, bool ignoreOrderBy = false)
         {
             if (_clause == "select")
             {
-                if (_skip != 0 || _count != 0)
+                if ((_skip != 0 || _count != 0) && (dialect is SqlServerDialect || dialect is SqliteDialect))
```

Comment:
Not good, we need to find a better solution, at least a `SupportXXX` in the dialect that would be false for MySql.